extension/resources/api-extension.json

```diff
@@ -7,7 +7,8 @@
   "triggers": [
     {
       "resourceTypeId": "payment",
-      "actions": ["Create", "Update"]
+      "actions": ["Create", "Update"],
+      "condition": "paymentMethodInfo is defined AND paymentMethodInfo(paymentInterface is defined) AND paymentMethodInfo(paymentInterface=\"ctp-adyen-integration\")"
```

Comment:
if `paymentMethodInfo(paymentInterface=\"ctp-adyen-integration\")` is true, then `paymentMethodInfo(paymentInterface is defined)` should be true, so we don't need the middle condition checking, isn't it?

Reply:
It would be too easy if it works like you suggested. Check here in the docs the orange box: https://docs.commercetools.com/api/projects/api-extensions#conditional-triggers

TBH I also tried your suggestion at first and it threw an error 😬 that's how I came to this condition in the end.